huaweicloud/services/cce/resource_huaweicloud_cce_node_attach.go

```diff
@@ -377,12 +386,12 @@ func resourceCCENodeAttachV3Create(ctx context.Context, d *schema.ResourceData,
 	}
 	addOpts.NodeList[0].Spec.Login = loginSpec
 
-	s, err := nodes.Add(nodeClient, clusterID, addOpts).ExtractAddNode()
+	s, err := nodes.Add(cceClient, clusterID, addOpts).ExtractAddNode()
 	if err != nil {
-		return fmtp.DiagErrorf("Error adding HuaweiCloud Node: %s", err)
+		return diag.Errorf("error adding node to : %s", err)
```

Comment:
adding node to CCE cluster?

Reply:
fixed and bring this file change to another PR.